specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationTextJobsTaskStatusRequest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "displayName": "Analyze chat",
+  "createdDateTime": "2022-04-01T15:00:45Z",
+  "expirationDateTime": "2022-04-02T15:00:45Z",
+  "jobId": "3e9e8518-492f-47f9-abd1-9a7468231086",
+  "lastUpdateDateTime": "2022-04-01T15:00:49Z ",
+  "status": "succeeded",
+  "tasks": {
+    "completed": 1,
+    "failed": 0,
+    "inProgress": 0,
+    "total": 1,
+    "items": [
+      {
+        "kind": "IssueResolutionSummarizationResults",
+        "lastUpdateDateTime": "2022-04-01T15:00:49Z",
+        "taskName": "analyze 1",
+        "status": "succeeded",
+        "results": {
+          "conversations": [
+            {
+              "id": "20220101meeting",
+              "summaries": [
+                {
+                  "aspect": "issue",
```

Comment:
Our experience from Opinion Mining shows that customers don't really understand the word "aspect". Might be worth thinking of another term (e.g. `category`).

Reply:
@mikaelsitruk do we want to change this?